.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
this again?

> Please add that in your global ignore file or in .git/info/exclude and remove it from the pr.

Reply:
In the next commit (63a808d), it was removed.